### PR TITLE
Fix transparent pixel

### PR DESCRIPTION
**Pull request by enmanuelmag**
I have changed the string that the "getChar" function returns for when the receiving pixel represents a transparent pixel, now it return a empty string. @mster

### CODE REVIEW

**Comment by mster** [comment]
@enmanuel-mag This is amazing! Thank you for contributing. I tested your change and it looks great. :)

Would you mind doing a one thing before I merge?
* This package uses StandardJS for linting. To make sure your code is in the right style, you can run the test command:
`npm test`. You can read more about StandardJS here https://standardjs.com/.

I appreciate you taking on this issue on such an immature package with no information about contributing. If you have any questions, or need some clarification, don't hesitate to ask!

**Comment by enmanuelmag** [comment]
> @enmanuel-mag This is amazing! Thank you for contributing. I tested your change and it looks great. :)
> 
> Would you mind doing a one thing before I merge?
> 
> * This package uses StandardJS for linting. To make sure your code is in the right style, you can run the test command:
>   `npm test`. You can read more about StandardJS here https://standardjs.com/.
> 
> I appreciate you taking on this issue on such an immature package with no information about contributing. If you have any questions, or need some clarification, don't hesitate to ask!

Hi! Oh that was new for me, I am new collaborating in open source project. I have already run the command you mentioned and reused the fixes. Sorry for the style change I had, I have an extension in vs code that changed caused by mistake that. Notice me if now is correct style

File: index.js [hunk 1]
```diff
@@ -87,20 +89,17 @@ function imscii (read, options) {
     } else if (value >= 125 && value < 150) {
       return ':'
     } else if (value >= 150 && value < 175) {
-      return ';'
```
Comment (mster) [inline index.js]: I think this got removed accidentally.

**Comment by mster** [comment]
No problem :) Thanks for making those changes.

Three more things and you're all set to merge!
1) The semicolon character `;` returned in `getChar` got removed accidentally. Could you add it back?
https://github.com/mster/imscii/pull/3/files#diff-168726dbe96b3ce427e7fedce31bb0bcL90

2) Would you mind squashing your commits down to a single commit? You can do this by running an interactive rebase. Here's a resource if you're new to it. https://blog.carbonfive.com/always-squash-and-rebase-your-git-commits/

3) Could you provide some information so that I can add you to the contributors list? For now, you can just reply as a comment and I'll add it.

    Example:
```json
"contributors": [
    "Michael Sterpka <michaelsterpka@gmail.com> (https://github.com/mster)",
  ],
```

**Comment by enmanuelmag** [comment]
Done, I've put together the commitments and here is the information you asked for:

```
"contributors": [
    "Enmanuel Magallanes <fmagalla@fiec.espol.edu.ec> (https://github.com/enmanuel-mag)",
  ]
```

**Comment by mster** [comment]
Perfect! I will have this merged soon and add you to contributors :) 

Thank you!

**Comment by mster** [comment]
@enmanuel-mag Merged and added you to the contributors! Thank you so much for your contribution :+1:

**Comment by enmanuelmag** [comment]
You welcome 😊

**Pull request merged**